test/unittests/execute_call_test.cpp

```diff
@@ -338,8 +338,8 @@ TEST(execute_call, imported_function_from_another_module)
     ASSERT_TRUE(func_idx.has_value());
 
     auto sub = [&instance1, func_idx](
-                   Instance&, std::vector<uint64_t> args, int) -> execution_result {
-        return fizzy::execute(*instance1, *func_idx, std::move(args));
+                   Instance&, span<const uint64_t> args, int) -> execution_result {
+        return fizzy::execute(*instance1, *func_idx, std::vector(args.begin(), args.end()));
```

Comment:
Why this?
It seems `args` passed to this lambda should live long enough on the caller side.

Reply:
A mistake.